src/Phinx/Migration/Manager.php

```diff
@@ -134,10 +134,11 @@ public function printStatus($environment, $format = null)
      *
      * @param string    $environment Environment
      * @param \DateTime $dateTime    Date to migrate to
+     * @param boolean   $dryRun      Output SQL rather than executing it
      *
      * @return void
      */
-    public function migrateToDateTime($environment, \DateTime $dateTime)
+    public function migrateToDateTime($environment, \DateTime $dateTime, $dryRun)
```

Comment:
Not so far as I can tell, because console arguments are only available at the Command level. It goes from Command to Manager to Environment, at which point we use it to determine whether to wrap the existing adapter in an instance of `DryRunWrapper`, which is the whole point. If you've got a better method of handling this, I'm definitely all for it.